apps/protocol-api/src/prisma/resolvers/Contribution.ts

```diff
@@ -274,6 +296,31 @@ export class ContributionCustomResolver {
     if (res.count !== 1) {
       throw `Wrong number of rows updated ${res.count} updateUserContribution`;
     }
+    const currentContribution = await prisma.contribution.findUnique({
+      where: { id: args.data.contributionId },
+    })
+    if (args.data.currentGuildId !== undefined) {
+
+      await prisma.contribution.update({
+        data: {
+          ...(args.data.guildId && {
+            guilds: {
+              delete: [{
+                guild_id_contribution_id: {
+                  contribution_id: args.data.contributionId,
+                  guild_id: args.data.currentGuildId,
+                  // guild_id: currentContribution.guilds[0].guild.id
```

Comment:
Probably good to remove

Reply:
Cool - removed this and also the now unused `findUnique` above it